docs/gameplay-component.md

```diff
@@ -12,21 +12,68 @@
 
 ## How is Genie structured?
 
+Genie is a modular framework which provides a set of reusable components (known as "screens") which are common across all BBC games, as well as various core "engines", which handle GEL buttons, layout and accessibility.
+
+The current screens are:
+
+Title
+How to Play
+Pause
+Loading
+Select
+Results
```

Comment:
These need bullet points in front of them as in MarkDown they will be positioned inline.